src/instructlab/configuration.py

```diff
@@ -647,21 +662,24 @@ class _train(BaseModel):
 
 
 class Config(BaseModel):
-    """Configuration for the InstructLab CLI."""
+    """Configuration for the InstructLab CLI.
+    Config options are defined by the respective subclasses and are loaded into a single 'Config' object here
+    Instantation of this object should be done via 'get_default_config()'
+    Note that values here can be overriden by a users 'config.yaml' or command line overrides in some cases
+    """
 
+    # chat configuration
     chat: _chat = Field(
         default_factory=_chat, description="Chat configuration section."
     )
+    # generate configuration
```

Comment:
These comments seem redundant since you've already explained what they are in the above docstrings + all of the other docstrings for each class

Reply:
Very fair point!